### ISSUE TITLE
Make pop-up prompts accessible to screen reader.

### ISSUE THREAD

**Issue by quachtina96** [LLK/scratch-gui#903]
### Expected Behavior
The screenreader should be able to read the message in all prompts (such as the one that pops upon deleting a sprite) so that the user knows what they are "OK"-ing or "cancel"-ing.

![image](https://user-images.githubusercontent.com/10423718/32919502-1160bc60-caf4-11e7-8349-9fc3e193b856.png)

### Actual Behavior

The screenreader reads "window, Cancel, button, OK, button" but does not say "Are you sure you want to delete this sprite"?

### Steps to Reproduce
Turn on screenreader (Command + F5 on Mac to turn on VoiceOver)
Navigate to a sprite's delete option and select it (or just click on the delete button on a sprite)

### Operating System and Browser

Mac OS 10.12.6 Chrome 62.0.3202.89, with VoiceOver screenreader

**Comment by quachtina96** [comment]
It looks like you can navigate left to read the text on the prompts, but the text isn't automatically read aloud. Doesn't look like it's something we have control over.

**Comment by thisandagain** [comment]
@quachtina96 We'd like to migrate all of these prompts over to our standard modal. You may want to look at https://github.com/LLK/scratch-gui/issues/340 before addressing this issue.
